CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   - `channel_group`
   - `ipv4_pim_sparse_mode`
   - `vlan_mapping`, `vlan_mapping_enable`
+  - `ipv4_acl_in`, `ipv4_acl_out`, `ipv6_acl_in`, `ipv6_acl_out`
```

Comment:
Please make sure to update the resource reference for these properties.

https://github.com/cisco/cisco-network-puppet-module/blob/develop/README.md#type-cisco_interface